investors/views.py

```diff
@@ -202,3 +211,64 @@ def perform_destroy(self, instance):
             extra={"saved_id": instance.pk, "by_user": self.request.user.pk},
         )
         super().perform_destroy(instance)
+
+class ViewedStartupPagination(pagination.PageNumberPagination):
+    """
+    Pagination class for recently viewed startups.
+    Default page size is 10, can be overridden via ?limit query parameter.
+    """
+    page_size = 10
+    page_size_query_param = "limit"
+    max_page_size = 50
+
+
+class ViewedStartupListView(generics.ListAPIView):
+    """
+    GET /api/v1/startups/viewed/
+    Retrieve a paginated list of recently viewed startups for the authenticated investor.
+    """
+    serializer_class = ViewedStartupSerializer
+    permission_classes = [IsAuthenticated, IsInvestor]
```

Comment:
The IsInvestor permission class is custom but not defined in this PR, which can cause confusion if missing. Include a comment or import for clarity.